tests/console/journalctl.pm

```diff
@@ -100,6 +107,25 @@ sub assert_test_log_entries {
     }
 }
 
+sub rotatelogs_and_verify {
+    push my @existing_rotations, split('\n', script_output('find /var/log/journal/ -regex ".*system\@.*" -o -regex ".*user-.*"'));
+    my $systemd_journal_birth = script_output 'stat -c %W /var/log/journal/$(cat /etc/machine-id)/system.journal';
+    my @errors;
+    assert_script_run('journalctl --rotate');
+    ($systemd_journal_birth >= script_output('stat -c %W /var/log/journal/$(cat /etc/machine-id)/system.journal')) &&
+      push @errors, 'New system.journal file has not been created!';
+    (scalar(@existing_rotations) >= scalar(split('\n', script_output('find /var/log/journal/ -regex ".*system\@.*" -o -regex ".*user-.*"')))) &&
```

Comment:
maybe:
```suggestion
    (@existing_rotations >= script_output('find /var/log/journal/ -regex ".*system\@.*" -o -regex ".*user-.*" | wc -l')) &&
```

Reply:
I am quite unsure about this part, as we want to compare ints/numbers. The nature of  `script_output` is to return strings and on top of that perl's equity operators differs between strings and ints. WDYT ?

Reply:
`scalar()` would also return a string, It just means that it isn't used in array context.

Perl evaluates any string to 0, if it isn't a number. 
```
> perl -e 'printf("%f\n", "asdf1.2e3");'
0.000000
> perl -e 'printf("%f\n", "1.2e3asdf");'
1200.000000
> perl -e 'printf("%f\n", "1.2e3");'
1200.000000
```
Same apply to int see: https://perldoc.perl.org/perlnumber#SYNOPSIS

So I would say it is safe to do it the `wc -l` way.